packages/jsii-rosetta/lib/commands/extract.ts

```diff
@@ -91,24 +91,25 @@ async function translateAll(
   snippets: IterableIterator<TypeScriptSnippet>,
   includeCompilerDiagnostics: boolean,
 ): Promise<TranslateAllResult> {
-  try {
-    const worker = await import('worker_threads');
-
-    return workerBasedTranslateAll(
-      worker,
-      snippets,
-      includeCompilerDiagnostics,
-    );
-  } catch (e) {
-    if (e.code !== 'MODULE_NOT_FOUND') {
-      throw e;
-    }
-    logging.warn(
-      'Worker threads not available (use NodeJS >= 10.5 and --experimental-worker). Working sequentially.',
-    );
+  if (!process.env.JSII_ROSETTA_NO_WORKER_THREADS) {
```

Comment:
Is this for windows compatibility?

Reply:
Oh durrr.... I didn't mean to commit that... It's when I was debugging, this was heavily interfering with my ability to set breakpoints in this part of the code :)